packaging/suse/amazon-ecs.service

```diff
@@ -10,7 +10,8 @@ Requires=network.target
 Type=simple
 ExecStartPre=/usr/sbin/amazon-ecs-init pre-start
 ExecStart=/usr/sbin/amazon-ecs-init start
-ExecStop=/usr/sbin/amazon-ecs-init pre-stop
+ExecStop=/usr/sbin/amazon-ecs-init stop
```

Comment:
are we expecting this change to be in here?

Reply:
cc @fierlion is this safe to go out?

Reply:
if yes, does this need changelog?

Reply:
yes this is a change unrelated to fd leak we decided to keep for this releaese.

Reply:
https://github.com/aws/amazon-ecs-init/issues/162 we'll address it with this issue